src/common/schemas/DBRP.yml

```diff
@@ -24,14 +24,3 @@
       description: Specify if this mapping represents the default retention policy for the database specificed.
     links:
       $ref: "./Links.yml"
-  oneOf:
```

Comment:
This part was breaking the CLI codegen. The `DBRP` schema now represents the DBRP response from the server, rather than trying to specify what the required inputs are as well as what the response looks like.

Reply:
Can we mark `bucketID`, `database`, and `retention_policy` as required to make up for this? It makes codegen nicer to have required things be marked.

Since `DBRP` is returned on GET requests, will `org` and `orgID` always be filled in? Could they also be marked as required?